scripts/prepare_transport_data_input.py

```diff
@@ -12,56 +12,87 @@
 import pandas as pd
 from _helpers import BASE_DIR
 
-# logger = logging.getLogger(__name__)
+logger = logging.getLogger(__name__)
+
+
+def _add_iso2_code_per_country_and_clean_data(df):
+
+    cc = coco.CountryConverter()
+    df["country"] = cc.pandas_convert(
+        series=pd.Series(df["Country"]), to="ISO2", not_found="not found"
+    )
+
+    df = df[df.country != "not found"]
+
+    # Drop region names where country column contains list of countries
+    df = df[df.country.apply(lambda x: isinstance(x, str))]
+
+    df = df.drop_duplicates(subset=["country"])
```

Comment:
Here we drop, should we sum instead? is there a reason?
It can be a TODO also, though a log info may be advisable if that happens

Reply:
The drop_duplicates function with keep='first' is only used to keep the WHO GHO list entries and delete the duplicate entries per country in the Wikipedia source. -> I have moved the operation to the vehicle function.

Would you still sum the values based on this information? Why would we do that?
In fact, the values per country are different more often than expected. For example, WHO-GHO reports 50.6 million registered vehicles in Vietnam, while Wikipedia reports 4.18 million. But I can't say which source (secondary or primary) is more valid or appropriate.
What would you recommend?